cirq/ops/gate_operation.py

```diff
@@ -61,6 +62,9 @@ def with_qubits(self, *new_qubits: raw_types.Qid) -> 'GateOperation':
     def with_gate(self, new_gate: raw_types.Gate) -> 'GateOperation':
         return new_gate.on(*self.qubits)
 
+    def has_gate_of_type(self, gate_type: Type[raw_types.Gate]) -> bool:
```

Comment:
I would remove this method, since it is redundant with `isinstance(op.gate, gate_type)`.

Reply:
done